src/actions.js

```diff
@@ -0,0 +1,26 @@
+export const inferLimit = function(response) {
+  const count = response.count;
+  const resultCount = response.results.length;
+
+  if (count > resultCount) {
```

Comment:
When a query has multiple pages, that expression will always be true for the first page response.
The times when it's not true, is on the last page, or when there are no results.